docs/antrea-ipam.md

```diff
@@ -122,12 +122,15 @@ IPPool annotation, or when the `AntreaIPAM` feature is disabled.
 
 #### Enable AntreaIPAM feature gate and bridging mode
 
-To enable flexible IPAM, you need to enable the `AntreaIPAM` feature gate for
-both `antrea-controller` and `antrea-agent`, and set the `enableBridgingMode`
-configuration parameter of `antrea-agent` to `true`.
+**Flexible IPAM** requires enabling the `AntreaIPAM` feature gate for both
+`antrea-controller` and `antrea-agent`, along with setting the `enableBridgingMode`
+parameter to `true` in `antrea-agent`.
 
-When Antrea is installed from YAML, the needed changes in the Antrea
-ConfigMap `antrea-config` YAML are as below:
+> **Version Compatibility Note:**  
+> The `AntreaIPAM` feature gate has been **enabled by default since v2.4**.
```

Comment:
Antrea v2.4

Reply:
Thanks @jianjuns, I noticed that some parts were added while others weren't. This seems to have almost no impact on the understanding of the document.

Reply:
No strong opinion here.